lib/tasks/import/local_custodian_codes_from_csv.rake

```diff
@@ -0,0 +1,6 @@
+namespace :import do
```

Comment:
I think the rake task and associated tests can be added in a commit after the `LocalCustodianCodesImporter` has been created. Nice self-contained commits that layer functionality on - makes it easier to review and track changes.

Reply:
Thank you for noticing this one, I will keep it in mind for the future, but perhaps will leave it in one commit this time as our docs advise to avoid rewriting commits unless really necessary?

Reply:
Are you referring to [these docs](https://gds-way.cloudapps.digital/standards/source-code.html#branching-merging-conventions)? I can see where it says that, but it's referring to commits that have already been pushed to `main`. Commits on unmerged branches, like this one, should "make liberal use of Git’s history rewriting features" 👍 

I agree that doc is a bit ambiguous. In this case though, we know that we "are very sure that no-one else will be affected by you rewriting the branch history", as there are no branches dependent on this one. I might raise a follow-up PR to the GDS Way to try and clarify this a bit (or you're welcome to!)

Reply:
I have split the commit @ChrisBAshton . Happy to raise follow-up PR 👍